### PR TITLE
WIP Normalizations

### PR DESCRIPTION
**Pull request by antoine-levitt**
Builds on top of 1D_fixes, not sure what the correct git etiquette is for this. Anyway, will rebase on top of master when 1D_fixes is merged, which should fix that (?)

Preparation for normalization fix, just some cosmetic fixes and docs right now, will next commit the correct normalization conventions (which should break everything)

### CODE REVIEW

**Comment by antoine-levitt** [comment]
OK, tests pass. @mfherbst I don't understand the code in `guess_gaussian_sad`, specifically the fact that the guess density is normalized to have total charge `charge_ionic(spec)`, so it doesn't have the correct number of electrons; shouldn't it be the number of valence electrons instead?

**Comment by mfherbst** [comment]
That's exactly what `charge_ionic` is, see https://github.com/JuliaMolSim/DFTK.jl/blob/master/src/core/Species.jl#L20. so it's the charge needed to compensate the "ionic change", which is defined in the PSP literature as the nuclear charge minus the core electrons as far as I know.

**Comment by antoine-levitt** [comment]
Ah sure, there's even `n_elec_valence(spec::Species) = charge_ionic(spec)`. My bad, thanks!

File: examples/silicon.jl [hunk 1]
```diff
@@ -3,8 +3,8 @@ using Printf
 using PyCall
 
 # Calculation parameters
-calculation_model = :lda
-kgrid = [4, 4, 4]        # k-Point grid
+calculation_model = :pbe
+kgrid = [1, 1, 1]        # k-Point grid
 supercell = [1, 1, 1]    # Lattice supercell
```
Comment (mfherbst) [inline examples/silicon.jl]: This looks like an accidential commit.

**Comment by mfherbst** [comment]
Why do you want to rename it? SAD = superposition of atomic densites ... that's kind of what it is, no?

**Comment by antoine-levitt** [comment]
It's a non-standard acronym (at least for me), and feels like an internal thing. Eg if we want other type of starting densities we can give them as keywords to `guess_density`.

File: examples/silicon.jl (hunk 1 above)
Comment (antoine-levitt) [reply examples/silicon.jl]: Woops, thanks!

**Comment by mfherbst** [comment]
I like the idea of the kwargs :smile:.

**Comment by antoine-levitt** [comment]
Hah, I didn't know about that! Looks like I still have some bad jokes to discover :p

Great to have this done!

**Pull request merged**